### DISCUSSION TITLE
Define and use of integer variables (YAML-files) - c7n-org

### DISCUSSION THREAD

**Discussion by m-wick** [cloud-custodian/cloud-custodian#7540] [General]
Hi guys,
I am facing an issue regarding c7n org and the account configuration file. Some integer variables must be defined in the variable section that will be used in a ASG resize policy as min, max and desired size. The issue is, that I am not able to hand over the value as integer.

"{min-size}" --> "Parameter validation failed:\nInvalid type for parameter MinSize, value: 0, type: <class 'str'>, valid types: <class 'int'>"
{min-size} --> "Parameter validation failed:\nInvalid type for parameter MinSize, value: {'min-size': None}, type: <class 'dict'>, valid types: <class 'int'>"

Also some other definitions with quotes and brackets did not fix this issue.

I saw that there is the field "value_type" for the filter section, but it looks like it can’t be used in the action section.

My question would be, how can I pass the variable as an integer value to the policies?

Best regards,
Marten

**Comment by leonardpahlke** [comment]
cc

**Comment by leonardpahlke** [comment]
see issue: https://github.com/cloud-custodian/cloud-custodian/issues/7553
